src/SFA.DAS.AssessorService.Domain/Extensions/DateTimeUtils.cs

```diff
@@ -63,5 +63,10 @@ public static string ToSfaShortestDateString(this DateTime? time)
             return time == null ? 
                 string.Empty : time.Value.ToString("dd MMM yy");
         }
+
+        public static string ToNumericalDateString(this DateTime? time)
+        {
+            return time == null ? string.Empty : time.Value.ToString("dd/MM/yyyy");
```

Comment:
You will probably be able to get away with `time.Value.ToString("d")`  https://docs.microsoft.com/en-us/dotnet/api/system.datetime.tostring?view=net-5.0

Reply:
I prefer the explicit "dd/MM/yyyy" to avoid any unintentional "en-US" defaults. But don't we have a SFA date format.
Yes. we have src\SFA.DAS.AssessorService.Domain\Extensions\DateTimeUtils.cs

Reply:
That's fair. @ben1stone may be worth either changing back or enforcing the en culture in the overload. Will leave it up to you, I know the rest of the dates are explicit in this util class.

Reply:
I am also concerned that we are being asked to format a date in the Assessor service UI in a different way.
I rather suspect that this is a CONTENT mis-align, as I don't see why we should be adding a new format type to display date.

Reply:
@alandburns In the main summary lists the date is 1 January 2021 but this was inside a dropdown so thought that was the distinction but I'll check with UX